src/tirith/cli.py

```diff
@@ -118,6 +118,10 @@ def __init__(self, prog="PROG") -> None:
                 print(formatted_result)
             else:
                 pretty_print_result_dict(result)
+
```

Comment:
@refeed can you please verify how this will impact the current terraform workflow step and if we are using the exit code correctly.

Reply:
@samriddhi99 here is my proposal for the exist codes: https://github.com/StackGuardian/tirith/issues/173
Let us please implement in this PR itself.